.azure/pipelines/ci.yml

```diff
@@ -716,6 +717,7 @@ stages:
         jobDisplayName: 'Tests: Helix x64'
         agentOs: Windows
         timeoutInMinutes: 240
+        # While isTestingJob is technically true, this job needs none of the normal installations nor test results.
```

Comment:
Maybe worth just renaming isTestingJob isAzdoTestingJob rather than putting this comment in all the helix yml?

Reply:
Done